pkg/golinters/lll.go

```diff
@@ -86,9 +86,27 @@ func getLLLIssuesForFile(filename string, maxLineLen int, tabSpaces string) ([]r
 
 	lineNumber := 1
 	scanner := bufio.NewScanner(f)
+	multiImportEnabled := false
 	for scanner.Scan() {
 		line := scanner.Text()
 		line = strings.ReplaceAll(line, "\t", tabSpaces)
+		if strings.HasPrefix(line, "import") {
+			if strings.HasSuffix(line, "(") {
+				multiImportEnabled = true
+			}
+
+			lineNumber++
+			continue
+		}
+		if multiImportEnabled {
+			if line == ")" {
+				multiImportEnabled = false
+			}
+
+			lineNumber++
+			continue
+		}
```

Comment:
```suggestion
	lineNumber := 1
	multiImportEnabled := false

	scanner := bufio.NewScanner(f)
	for scanner.Scan() {
		line := scanner.Text()
		line = strings.ReplaceAll(line, "\t", tabSpaces)

		if strings.HasPrefix(line, "import") {
			multiImportEnabled = strings.HasSuffix(line, "(")
		}

		if multiImportEnabled {
			if line == ")" {
				multiImportEnabled = false
			}

			lineNumber++
			continue
		}
```

Reply:
I removed `if`, but left
```go
lineNumber++
continue
```
for `if strings.HasPrefix(line, "import") {` because we want skip single line import too.

Is it correct?

Reply:
yes sorry